cypress/integration/capture.spec.js

```diff
@@ -194,6 +196,45 @@ describe('Event capture', () => {
         })
     })
 
+    describe('capturing performance', () => {
+        it('sends performance timing with the pageview when enabled', () => {
+            given('options', () => ({ capture_pageview: true, capture_performance: true }))
+            start()
+
+            // Pageview will be sent immediately
+            cy.wait('@capture').its('request.headers').should('deep.equal', {
+                'Content-Type': 'application/x-www-form-urlencoded',
+            })
+            cy.get('@capture').should(({ request }) => {
+                const captures = getBase64EncodedPayload(request)
+
+                expect(captures['event']).to.equal('$pageview')
```

Comment:
Note: This looks inconsistent if we're accessing some properties via [] and others via . - pick one.